content/pages/work-with-us.html

```diff
@@ -1,59 +1,87 @@
+- **Usability research and data-driven assessments.** Conducting comprehensive research to understand public needs, employing data to enhance the quality and effectiveness of services.
 
-The following table is a rough estimate of resources for the kinds of projects we engage in. We can customize our support to help wherever you are in the process of transforming your digital services.
+Some of our typical offerings include:
 
-| Offering      | Benefits  | Approx time |
-| ----------- | ----------- | ----------- |
-|[Strategy Development]({{ "/strategy-development/" | url }})   | Evaluate your options to get actionable recommendations to achieve your goals.  | 3 months  |
-|[Vendor Acquisition Support]({{ "/vendor-acquisition-support/" | url }})   |Learn what you need to buy and get help issuing and managing a procurement, from RFI to award.   |6 months - 1.5 years|
-|[Website Modernization]({{ "/website-modernization/" | url }})  | Reimagine your complete website experience. Prototype, implement improvements, and get the tools and coaching to manage your new site.  |3 months - 1 year  |
-|[Digital Service Transformation]({{ "/digital-service-transformation/" | url }})  |Research, prototype, launch, and manage your digital products from start to finish. Get support and coaching on how to continually improve upon your success.   | 2.5 years|
+### Strategy Development
 
+The first step toward success starts with a viable strategy. 18F strategy development teams help you wrangle complex service problems and develop a plan to achieve your goals. 
 
-To take the next step, email us at <a href="mailto:inquiries18F@gsa.gov">inquiries18F@gsa.gov</a> and tell us what your needs are.
-{% endcapture %}
+To create your strategy, we assemble a team with the right expertise to help you get a hold of your challenge. We draw on our experience and technical skills to develop a strategy based on rigorous research. We prototype solutions and test them with your stakeholders and users. These steps will help ensure that the strategy we recommend is viable and will satisfy both user needs and business goals, so you feel confident about each step.
 
-{% capture mission-path %}
-## Choose the path that’s best for your mission and your users
+### Vendor Acquisition Support
 
-We don’t offer one-size-fits-all solutions. Instead, we work with you to design the right path for your organization and ensure that you feel confident about each step. We believe in “demos not memos” and everything we do is intended to be a step toward working software.
+Some software projects are the right size and shape for 18F to build with you; others will require you to hire an outside vendor. We can help you hire a vendor that can respond to your changing needs and deliver working software to you on a regular basis. We’ll help you issue and manage a procurement, from RFI to post-award.
```

Comment:
concern: This buries the lede. Also, something about "for 18F to build with you;" feels clunky, and the semicolon is unforced. 

change: Start the paragrpah with something that communicates the intent from the start, in the spirit of "18F can help you hire a vendor to build your project." The line about "some projects are the right size, while others need outside support" can come after.

Reply:
@durland18F - thoughts?